src/Tribe/Views/V2/Assets.php

```diff
@@ -528,6 +528,17 @@ public function should_enqueue_frontend() {
 			return $this->should_enqueue_frontend;
 		}
 
+		/**
+		 * Checks whether the page is being viewed in Elementor preview mode.
+		 * 
+		 * @since TBD
+		 * 
+		 * @return bool $should_enqueue Should the frontend assets be enqueued.
+		 */
+		if ( ! ( ! defined( 'ELEMENTOR_PATH' ) || empty( ELEMENTOR_PATH ) ) && \Elementor\Plugin::$instance->preview->is_preview_mode() ) {
```

Comment:
Are we sure that we should _always_ load this in The Elementor preview? This bypasses **all** the checks in Template_Bootstrap

Reply:
If so - the logic could be simplified a lot.
```suggestion
		if (
			defined( 'ELEMENTOR_PATH' )
			&& ! empty( ELEMENTOR_PATH )
			&& \Elementor\Plugin::$instance->preview->is_preview_mode()
        ) {
```

Reply:
...and the call to `is_preview_mode` can probably be boiled down to (for our purposes)

`if ( isset( $_GET[ 'elementor-preview' ] ) && $_GET[ 'elementor-preview' ] === get_the_ID() )`

removing the requirement for calling a third-party class?

Sorry, to be clearer, try:
```
if (
	defined( 'ELEMENTOR_PATH' )
	&& ! empty( ELEMENTOR_PATH )
	&& isset( $_GET[ 'elementor-preview' ] )
	&& $_GET[ 'elementor-preview' ] === get_the_ID()
) {
```

Reply:
@Camwyn we need to load it on the elementor preview when the event view widget is added to the page.

Reply:
I've updated the conditional but haven't added the last one i.e. `$_GET[ 'elementor-preview' ] === get_the_ID()`. `isset( $_GET[ 'elementor-preview' ] )` works as expected.

Reply:
Yeah, the `$_GET[ 'elementor-preview' ] === get_the_ID()` part was pulled directly from `is_preview_mode()` this is really nice and simple now 😄